schema/field/annotation.go

```diff
@@ -19,6 +19,30 @@ type Annotation struct {
 	//	}
 	//
 	StructTag map[string]string
+
+	// ID defines a multi-field schema identifier. Note,
+	// the annotation is valid only for edge schemas.
+	//
+	//	func (TweetLike) Annotations() []schema.Annotation {
+	//		return []schema.Annotation{
+	//			field.ID("user_id", "tweet_id"),
+	//		}
+	//	}
+	//
+	ID []string
+}
+
+// ID defines a multi-field schema identifier. Note, the
+// annotation is valid only for edge schemas.
+//
+//	func (TweetLike) Annotations() []schema.Annotation {
+//		return []schema.Annotation{
+//			field.ID("user_id", "tweet_id"),
+//		}
+//	}
+//
+func ID(field string, fields ...string) *Annotation {
```

Comment:
Agree, but I prefer the name `ID`, because this present the identifier of an entity. The composite word fits better for defining "composite primary keys"